tasks/tests/tests_view_edit_task.py

```diff
@@ -0,0 +1,65 @@
+from tasks.forms import TaskForm
+from tasks.models import Task
+import pytest
+
+
+@pytest.mark.django_db
+class TestViewEditTask:
+    def test_no_authenticated_edit_task(self, client, task_1):
+        response = client.get(f'/tasks/edit/{task_1.id}')
+        assert response.status_code == 200
+        assert 'auth' not in response.context.keys()
+
+    def test_employee_edit_task(self, client, employee_1, task_1):
+        client.login(username=employee_1.user.username, password='password')
+        response = client.get(f'/tasks/edit/{task_1.id}')
+        assert response.status_code == 200
+        assert not response.context['auth']
+
+    def test_manager_edit_task(self, client, manager_1, task_1):
+        client.login(username=manager_1.user.username, password='password')
+        response = client.get(f'/tasks/edit/{task_1.id}')
+        assert response.status_code == 200
+        assert response.context['auth']
+
+    def test_view_edit_task_form(self, client, manager_1, task_1):
+        client.force_login(manager_1.user)
+        response = client.get(f'/tasks/edit/{task_1.id}')
+        assert response.status_code == 200
+        assert isinstance(response.context.get('form'), TaskForm)
+
+    def test_load_edit_view(self, client, manager_1, task_2, task_2_data):
+        client.force_login(manager_1.user)
+        response = client.get(f'/tasks/edit/{task_2.id}')
+        assert response.status_code == 200
+        form_load_data = response.context['form'].initial
+        assert form_load_data['title'] == task_2_data['title']
+        assert form_load_data['assignee'] == task_2_data['assignee']
+        assert form_load_data['created_by'] == task_2_data['created_by']
+        assert form_load_data['description'] == task_2_data['description']
+        assert str(form_load_data['status']).upper() == task_2_data['status']
+        assert str(form_load_data['priority']).upper() == task_2_data['priority']
```

Comment:
What about using `test2` instead of `task2_data`?
Something like this
```suggestion
    def test_load_edit_view(self, client, manager_1, task_2):
        client.force_login(manager_1.user)
        response = client.get(f'/tasks/edit/{task_2.id}')
        assert response.status_code == 200
        form_load_data = response.context['form'].initial
        assert form_load_data['title'] == task_2.title
        assert str(form_load_data['assignee']) == str(task_2.assignee)
        assert str(form_load_data['created_by']) == str(task_2.created_by)
        assert str(form_load_data['description']) == str(task_2.description)
        assert str(form_load_data['status']).upper() == str(task_2.status).upper()
        assert str(form_load_data['priority']).upper() == str(task_2.priority).upper()
```

Reply:
OK, I applied the changes, but I still need the task_2_data for the test 'test_edit_task_valid' cause I send it as a parameter for the form, and it needs to be an object with 'items()', the task model doesn't have this attribute